2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
+
+### Shipping Timeline
+After countless back and forths with our FCC testing counterparts, we have an end in sight. Seeed Studio expects us to have results from FCC testing back by Friday, November 27th. Once FCC certification is passed, sourcing the components will take up to 6 weeks. Then we can start producing in batches of 500/week. This puts the first shipments out in mid-January. Feel free to get in touch if the new date is an issue for you (support@tessel.io).
+
+### Get excited
+
+We've already built most of our Tessel Start experience for the Tessel 2 - [give yourself a tour](//tessel.io/t2-start)! We've also released a set of preliminary Tessel 2 docs to [tessel.io/docs](https://tessel.io/docs/home).
+
+### Get involved
+
+We're also excited to have finished open sourcing literally all of our repos, including (at long last) our website, [tessel.io](https://github.com/tessel/tessel.io). Now, if you find something broken there you can file an issue or PR instead of reporting over Twitter.
```

Comment:
well, there is a bit of a higher bar than that to be added to the team list - not all contributors are Team Members (see tessel.io/project). I like this phrasing, though.